packages/web/src/components/create-playlist/PlaylistForm.tsx

```diff
@@ -255,18 +184,29 @@ const PlaylistForm = ({
           />
         </div>
       </div>
-
-      {isEditMode ? (
-        <EditPlaylistActions
-          isAlbum={isAlbum}
-          onCancel={onCancel}
-          onDelete={onDelete}
-          onSave={onSave}
-          disabled={hasSubmitted}
-        />
-      ) : (
-        <CreatePlaylistActions onSave={onSave} disabled={hasSubmitted} />
-      )}
+      <div className={styles.actionsWrapper}>
+        {isEditMode ? (
+          <EditActions
+            deleteText={
+              isAlbum
+                ? messages.deleteAlbumButtonText
+                : messages.deletePlaylistButtonText
+            }
+            saveText={messages.editPlaylistButtonText}
```

Comment:
not familiar with full context, but can these message objects be defined in the editactions component file?

Reply:
ahh clutch, sg